includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
OK guys, I had today another idea. What if we added in the Corporate Footer dropdown those wikis which have this footer enabled (`$wgEnableCorporateFooterExt`)? It's strange that wiki can be included in the dropdown on other pages, but actually it doesn't have this dropdown itself. What do you think? It would also lead to enabling Corporate Footer on pt-br hub/corp page.